Arrowgene.Ddon.Server/GameLogicSetting.cs

```diff
@@ -103,6 +103,9 @@ public class GameLogicSetting
         /// </summary>
         [DataMember(Order = 15)] public uint PawnCatchupLvDiff { get; set; }
 
+        [DataMember(Order = 16)] public uint JobLevelMax { get; set; }
```

Comment:
There's already a JobLevelMax (although not enforced) in the class LoginServerSetting, which gets sent to the client through a L2CGetLoginSettingsRes. These two should be unified

Reply:
I take it this has been solved, right?